app/views/nsm/steps/view_claim/work_items.html.erb

```diff
@@ -8,67 +8,76 @@
         time = work_items_for_type.sum(&:time_spent)
         net_cost = work_items_for_type.sum(&:total_cost)
         total_cost += net_cost
         [
           { text: t("summary.nsm/cost_summary/work_items.#{work_type.to_s}"), width: 'govuk-!-width-one-quarter' },
-          { text: format_period(time), numeric: true },
+          { text: format_period(time, style: :minimal_html), numeric: true },
           { text: NumberTo.pounds(net_cost), numeric: true },
         ]
       end
 
       foot = [
         { text: t(".total"), width: 'govuk-!-width-one-quarter' },
         { text: '', numeric: true },
-        { text: NumberTo.pounds(total_cost), numeric: true },
+        { text: govuk_visually_hidden(t('.net_cost_description')) + NumberTo.pounds(total_cost), numeric: true },
       ]
 
 
       govuk_table do |table|
         table.with_head(rows: [head])
-        table.with_body(rows: rows.sort_by { _1.dig(0, :text) }, first_cell_is_header: true)
+        table.with_body(rows: rows, first_cell_is_header: true)
         table.with_foot(rows: foot)
       end
     %>
   <% end %>
 
   <%
-    pagy, work_items = pagy(@work_items)
-    head = [
-      { text: t('.item'), width: 'govuk-!-width-one-quarter' },
-      { text: t('.time'), numeric: true },
-      { text: t('.uplift'), numeric: true },
-      { text: t('.net_cost'), numeric: true },
-      { text: t('.action'), numeric: true }
-    ]
-    work_items.group_by(&:completed_on).each do |completed_on, work_items_for_date| %>
-    <%=
-      rows = work_items_for_date.map do |work_item|
-        link = link_to(t('.view'), item_nsm_steps_view_claim_path(id: current_application.id, item_type: :work_item, item_id: work_item.id, page: pagy.page), data: { turbo: 'false' })
+    pagy, work_items = pagy_array(@work_items, items: 100)
+    head = {
+      'line_item' => false,
+      'item' => false,
+      'date' => false,
+      'fee_earner' => false,
+      'time' => [],
+      'uplift' => [],
+      'net_cost' => []
+    }
+  %>
+  <%=
+    rows = work_items.each_with_index.map do |work_item, index|
+      item_with_link = link_to(t("summary.nsm/cost_summary/work_items.#{work_item.work_type.to_s}"), item_nsm_steps_view_claim_path(id: current_application.id, item_type: :work_item, item_id: work_item.id, page: pagy.page), data: { turbo: 'false' })
 
-        [
-          t("summary.nsm/cost_summary/work_items.#{work_item.work_type.to_s}"),
-          { text: format_period(work_item.time_spent), numeric: true },
-          { text: NumberTo.percentage(work_item.uplift.to_f, multiplier: 1), numeric: true },
-          { text: NumberTo.pounds(work_item.total_cost), numeric: true },
-          { text: link, numeric: true }
-        ]
-      end
+      [
+        { header: true, text: index + 1, numeric: false},
+        { header: true, text: item_with_link, numeric: false},
+        { text: work_item.completed_on&.strftime("%-d %b %Y"), numeric: false},
```

Comment:
```suggestion
        { text: work_item.completed_on&.to_fs(:stamp), numeric: false},
```

Reply:
`date.to_fs(:stamp)` is giving me '22 July 2024' whereas We wanted to show as '22 Jul 2024'. I went through rails api and found out `date.to_fs(:rfc822)` this works.
[Api Doc Link](https://api.rubyonrails.org/classes/Date.html#method-i-to_formatted_s)

Reply:
I'd query that with Mary/Chris - I'd be somewhat surprised if we were deliberately choosing inconsistent date formatting across our service

Reply:
Added custom formatting named `:short_stamp` in initializers.